build.gradle

```diff
@@ -28,25 +28,16 @@ allprojects {
         maven { url 'https://mapbox.bintray.com/mapbox' }
 
         maven {
-            credentials {
-                username project.properties['mapboxMavenUser'] ?: ""
-                password project.properties['mapboxMavenToken'] ?: ""
-            }
+            url 'https://api.mapbox.com/downloads/v2/releases/maven'
```

Comment:
Are Android public docs updated to reflect it?

Reply:
Not yet, working on it, thanks for reminder

Reply:
Do you think its a blocker for teaser?